### ISSUE TITLE
UIImage+WebP.h import error 

### ISSUE THREAD

**Issue by eyu0415** [SDWebImage/SDWebImageWebPCoder#26]
recently SDWebImage v5.3.1 doesn't has file called 'SDWebImage.h'

**Comment by dreampiggy** [comment]
Build Success.

![image](https://user-images.githubusercontent.com/6919743/68605723-d6726880-04e7-11ea-92c3-302cb6ce6d64.png)


You can provide your detail use case using SDWebImage's template like this, for me to investigate the issues.

### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
The SDWebImage 5.3.1 via CocoaPods or Carthage can successfuly import the `SDWebImage.h`. You may have some wrong configuration, or you use something not standard installation guide. (Does this comes from reacti-native-fast-image) ?
